embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/CurrentSessionSpanImpl.kt

```diff
@@ -39,7 +37,7 @@ internal class CurrentSessionSpanImpl(
     /**
      * The span that models the lifetime of the current session or background activity
      */
-    private val sessionSpan: AtomicReference<Span?> = AtomicReference(null)
+    private val sessionSpan: AtomicReference<EmbraceSpan?> = AtomicReference(null)
```

Comment:
Ah nice. I didn't even notice this since this was written so long ago! It made a bit more sense when it was part of SpansServiceImpl, but now that this is a separate object, it sticks out lol. Cool cool.